core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/dot-uve-toolbar.component.ts

```diff
@@ -91,6 +104,42 @@ export class DotUveToolbarComponent {
 
     protected readonly date = new Date();
 
+    defaultDevices = DEFAULT_DEVICES;
+
+    // IF YOU DONT SEE ANY COMMENTS EXPLAINING THE CODE, PLEASE LEAVE ME A COMMENT, BECAUSE WE NEED DOCS OF THIS
+    handleViewParamsEffect = effect(
+        () => {
+            const { device: deviceInode, orientation } = this.#store.viewParams();
+
+            // So we dont open the editor in a device
+            if (!this.#store.$isPreviewMode() && (deviceInode || orientation)) {
+                this.#store.patchViewParams({ device: null, orientation: null });
```

Comment:
Doesn't this generate a loop any time?
Maybe it should be in an untracked()

Reply:
No it doesn't but I haven't tested this code properly, so Im sure this can be better